Quadriga/src/main/java/edu/asu/spring/quadriga/domain/impl/workspace/TextFile.java

```diff
@@ -0,0 +1,63 @@
+package edu.asu.spring.quadriga.domain.impl.workspace;
+
+import org.springframework.stereotype.Component;
+
+import edu.asu.spring.quadriga.domain.workspace.ITextFile;
+
+@Component
+public class TextFile implements ITextFile {
+
+    private String fileName;
+    private String projectId;
+    private String workspaceId;
+    private String refId;
+    private String fileContent;
```

Comment:
I think your refId should be the id/handle that is used in submitted networks for that text and an actually id attribute is missing.